client/views/room/contextualBar/UserInfo/index.js

```diff
@@ -81,9 +81,10 @@ export const UserInfo = React.memo(function UserInfo({
 				<Info>{username}</Info>
 			</>}
 
-			<Label>{t('Last_login')}</Label>
-			<Info>{lastLogin ? timeAgo(lastLogin) : t('Never')}</Info>
-
+			{lastLogin && <>
+				<Label>{t('Last_login')}</Label>
+				<Info>{timeAgo(lastLogin)}</Info>
```

Comment:
@Darshilp326 We need to find a way to verify when the user never did log in and when he's not allowed to see the last login

Reply:
Not able to get your point! :worried:

Reply:
@dougfabris Can you explain your point one more time so I can solve the issue?

Reply:
I mean, the code before checks if we don't get `lastLogin`'s value we'll show 'Never'. At this point, we should find a way to verify if we're receiving an empty value or if we shouldn't show the value because of the permission

Reply:
I mean, the code before checks if we don't get `lastLogin`'s value we'll show 'Never'. At this point, we should find a way to verify if we're receiving an empty value or if we shouldn't show the value because of the permission

Reply:
Ok got your point!

Reply:
That case would occur when the admin adds an user and that user has not logged in. Am I right?

Reply:
Sorry. Am I correct?

> That case would occur when the admin adds a new user and that user has not logged in. Am I right?